Add note guidelines for additional distributions (#80389)
ansible/ansible: docs/docsite/rst/installation_guide/installation_distros.rst

Installing Ansible on specific operating systems
================================================

> .. note:: These instructions are provided by their respective communities. Any bugs/issues should be filed with that community to update these instructions. Ansible maintains only the pip install instructions.

Reviewer: I'm pretty sure there must be an empty line after the title.
```suggestion

.. note:: These instructions are provided by their respective communities. Any bugs/issues should be filed with that community to update these instructions. Ansible maintains only the ``pip install`` instructions.
```

Author: I believe you are correct, now looking at the reStructured syntax in comparison to markdown.

The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.

The following instructions will guide you through installing the ``ansible`` package with your preferred distribution's package manager.

> .. note:: For maintainers wanting to add distributions not mentioned within the guide, install instructions are included here only for distributions with reasonably up to date version of ansible and follow Ansible policies and version constraints. The distribution MUST ensure that ansible-core and ansible are synced.Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the Ansible Packaging <https://matrix.to/#/#packaging:ansible.com_ matrix room.

Reviewer: This seems to have a `>` and backticks missing.
```suggestion
.. note:: For maintainers wanting to add distributions not mentioned within the guide, install instructions are included here only for distributions with reasonably up to date version of ``ansible`` and follow Ansible policies and version constraints. The distribution MUST ensure that ``ansible-core`` and ``ansible`` are synced. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.
```

Author: >This seems to have a `>` and backticks missing.

Will incorporate syntax corrections and be cognizant of markup syntax going forward.

Installing Ansible on specific operating systems
================================================

.. note:: These instructions are provided by their respective communities. Any bugs/issues should be filed with that community to update these instructions. Ansible maintains only the ``pip install`` instructions.

The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.

The following instructions will guide you through installing the ``ansible`` package with your preferred distribution's package manager.

> .. note:: For maintainers who wish to add distributions to this guide, installation instructions are included here only for distributions with a reasonably up-to-date version of ``ansible``. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that their build system allows. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.

Reviewer 2: ```suggestion
.. note:: For maintainers who wish to add distributions to this guide, installation instructions are included here only for distributions with a reasonably up-to-date version of ``ansible``. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that the distribution build system allows. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.
```

Author: I need to squash the vagueness is my writing, direct reference always sounds better. Thanks :)

Reviewer 2: It was [my suggestion](https://github.com/ansible/ansible/pull/80389/#issuecomment-1499239076) to use `they`, but then I realized that pronoun was confusing :).

> .. note:: For maintainers wanting to add distributions not mentioned within the guide, install instructions are included here only for distributions with reasonably up to date version of ``ansible`` and follow Ansible policies and version constraints. The distribution MUST ensure that ``ansible-core`` and ``ansible`` are synced. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.

Reviewer 3: ```suggestion
.. note:: For maintainers wanting to add distributions not mentioned within the guide, install instructions are included here only for distributions with reasonably up to date version of ``ansible`` and follow Ansible policies and version constraints. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.
```

Author: wording selection really does make the difference, I'll keep it concise in the future.

> .. note:: For maintainers wanting to add distributions not mentioned within the guide, install instructions are included here only for distributions with reasonably up to date version of ``ansible`` and follow Ansible policies and version constraints. The distribution MUST ensure that ``ansible-core`` and ``ansible`` are synced. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.

Reviewer 3: ```suggestion
.. note:: For maintainers wanting to add distributions not mentioned within the guide, install instructions are included here only for distributions with reasonably up-to-date version of ``ansible`` and follow Ansible policies and version constraints. The distribution MUST ensure that ``ansible-core`` and ``ansible`` are synced. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.
```

> .. note:: For maintainers who wish to add distributions to this guide, installation instructions are included here only for distributions with a reasonably up-to-date version of ``ansible``. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that the distribution build system allows. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ matrix room.

Reviewer 2: ```suggestion
.. note:: For maintainers who wish to add distributions to this guide, installation instructions are included here only for distributions with a reasonably up-to-date version of ``ansible``. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that the distribution build system allows. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ Matrix room.
```
